src/lib/transaction_snark_scan_state/inputs.ml

```diff
@@ -95,6 +103,6 @@ module type S = sig
   end
 
   module Transaction_witness :
-    Coda_pow.Transaction_witness_intf
-    with type sparse_ledger := Sparse_ledger.t
+    Transaction_witness_intf
+    with type sparse_ledger := Sparse_ledger.Stable.V1.t
```

Comment:
As far as I see, there isn't a `bin_io` requirement for the type `sparse_ledger`, so we can use `Sparse_ledger.t`.